packages/ckeditor5-alignment/src/alignmentediting.js

```diff
@@ -86,11 +79,12 @@ export default class AlignmentEditing extends Plugin {
 
 // Prepare downcast conversion definition for inline alignment styling.
 // @private
-function buildDowncastInlineDefinition( optionNames ) {
+function buildDowncastInlineDefinition( options ) {
+	const optionNames = options.map( option => option.name );
```

Comment:
Maybe we could inline it?

Reply:
There are two places in this function using it. Does it make sense to inline it and repeat the same operation twice?

Reply:
The first one uses a list of names, and the second one is iterating itself